src/components/radiobutton/radio-button-group/radio-button-group.directive.ts

```diff
@@ -0,0 +1,91 @@
+import { AfterContentInit, ContentChildren, Directive, EventEmitter, ExistingProvider, forwardRef, Input, OnDestroy, Output, QueryList } from '@angular/core';
+import { RadioButtonComponent } from '../radiobutton.component';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+/**
+ * Needed for Reactive forms so there is a top level element
```

Comment:
This kind of comment is a documentation comment, so I'd suggest not using it for developer comments in case we do some automatic processing of doc comments in the future.

Reply:
removed